lib/ProductOpener/Images.pm

```diff
@@ -1213,6 +1213,12 @@ sub process_image_move ($user_id, $code, $imgids, $move_to, $ownerid) {
 
 				delete $product_ref->{images}{$imgid};
 
+				# Check if any selected image's imgid equals the deleted imgid
```

Comment:
We want to check if the value of the "imgid" field of a selected field is equal to $imgid.

To verify the problem, see steps: https://github.com/openfoodfacts/openfoodfacts-server/issues/9793#issuecomment-1994223630